### DISCUSSION TITLE
shippng to the us?

### DISCUSSION THREAD

**Discussion by ecto1a2003** [gpstar81/GPStar-proton-pack#228] [General]
hi the ultimate kit seems out of stock at sonofagunn is there another source?

**Comment by gpstar81** [comment]
> hi the ultimate kit seems out of stock at sonofagunn is there another source?

It should be working again if you try sonofagunn props. They had a minor setting error on that bundle which is now fixed. Just a fyi they won’t be resuming shipping until sometime this month.
